dask_expr/_reductions.py

```diff
@@ -35,8 +35,13 @@ class Chunk(Blockwise):
 
     _parameters = ["frame", "kind", "chunk", "chunk_kwargs"]
 
-    def operation(self, df, *args, **kwargs):
-        return self.chunk(df, *args, **kwargs)
+    @staticmethod
+    def chunk_operation(op, *args, **kwargs):
+        return op(*args, **kwargs)
+
+    @property
+    def operation(self):
+        return functools.partial(self.chunk_operation, self.chunk)
```

Comment:
Hmm, I suppose we could just return `self.chunk`.

Reply:
I believe this is still problematic.

I suggest to add tests like I described in https://github.com/dask-contrib/dask-expr/issues/323#issuecomment-1753252862 to the project since catching this stuff in reviews alone is difficult

Reply:
The `partial` is still binding to `self.chunk_operation`, i.e. when serializing the `partial` object we have to serialize the entire instance